src/pages/CreateAssignment.vue

```diff
@@ -424,6 +429,138 @@ const scrollToError = (elementId) => {
   }, 100);
 };
 
+const hasAssignmentChanges = () => {
+  const original = existingAdministrationData.value;
+  const current = state;
+
+  // Compare name
+  if (current.administrationName !== original.name) {
+    return true;
+  }
+
+  // Compare dates - normalize to compare only date part (ignore time)
+  const normalizeDate = (date) => {
+    if (!date) return null;
+    const d = new Date(date);
+    return new Date(d.getFullYear(), d.getMonth(), d.getDate()).getTime();
+  };
+
+  const originalStartDate = normalizeDate(original.dateOpened);
+  const currentStartDate = normalizeDate(current.dateStarted);
+  if (originalStartDate !== currentStartDate) {
+    return true;
+  }
+
+  const originalEndDate = normalizeDate(original.dateClosed);
+  const currentEndDate = normalizeDate(current.dateClosed);
+  if (originalEndDate !== currentEndDate) {
+    return true;
+  }
+
+  // Compare sequential
+  if (current.sequential !== original.sequential) {
+    return true;
+  }
+
+  // Compare testData
+  if (isTestData.value !== original.testData) {
+    return true;
+  }
+
+  // Compare legal/consent fields
+  const originalConsent = original.legal?.consent ?? null;
+  const originalAssent = original.legal?.assent ?? null;
+  const originalAmount = original.legal?.amount ?? '';
+  const originalExpectedTime = original.legal?.expectedTime ?? '';
+
+  if (current.consent !== originalConsent) {
+    return true;
+  }
+  if (current.assent !== originalAssent) {
+    return true;
+  }
+  if (current.amount !== originalAmount) {
+    return true;
+  }
+  if (current.expectedTime !== originalExpectedTime) {
+    return true;
+  }
```

Comment:
We can get rid of this block, we used consent form for one site early on but now all sites will handle that themselves. We need to remove the old code about consent/assent.